puppet/lib/puppet/indirector/facts/puppetdb.rb

```diff
@@ -62,7 +62,7 @@ def find(request)
             Http.multi_arity_get(http_instance,
```

Comment:
Thanks, @mullr.  @adrienthebo, @rlinehan - since we're in here already, what would you think about collapsing out the `Http.multi_arity_get` functionality as part of this PR given that?